pages/Transfer.qml

```diff
@@ -306,6 +274,54 @@ Rectangle {
           }
       }
 
+      RowLayout{
+          StandardButton {
+              id: resolveButton
+              anchors.left: parent.left
+              width: 80
+              text: qsTr("Resolve") + translationManager.emptyString
+              visible: isValidOpenAliasAddress(addressLine.text)
+              enabled : isValidOpenAliasAddress(addressLine.text)
+              onClicked: {
+                  var result = walletManager.resolveOpenAlias(addressLine.text)
+                  if (result) {
+                      var parts = result.split("|")
+                      if (parts.length == 2) {
+                          var address_ok = walletManager.addressValid(parts[1], appWindow.persistentSettings.testnet)
+                          if (parts[0] === "true") {
+                              if (address_ok) {
+                                  addressLine.text = parts[1]
+                                  addressLine.cursorPosition = 0
```

Comment:
Looks like the property `cursorPosition` was removed when `addressLine` switched from `LineEdit` to `LineEditMulti` in the "dark theme" PR.

Reply:
Good catch, thanks